pkg/registration/supply_chain_factory.go

```diff
@@ -131,34 +140,15 @@ func (f *SupplyChainFactory) buildQuotaSupplyBuilder() (*proto.TemplateDTO, erro
 	return nodeSupplyChainNodeBuilder.ConnectsTo(vmQuotaExternalLink).Create()
 }
 
-func (f *SupplyChainFactory) buildNodeSupplyBuilder() (*proto.TemplateDTO, error) {
-	nodeSupplyChainNodeBuilder := supplychain.NewSupplyChainNodeBuilder(proto.EntityDTO_VIRTUAL_MACHINE)
-	nodeSupplyChainNodeBuilder = nodeSupplyChainNodeBuilder.
-		Sells(vCpuTemplateComm).
-		Sells(vMemTemplateComm).
-		Sells(vmpmAccessTemplateComm).
-		// TODO we will re-include provisioned commodities sold by node later.
-		//Sells(cpuProvisionedTemplateComm).
-		//Sells(memProvisionedTemplateComm)
-		Sells(cpuAllocationTemplateCommWithKey).
-		Sells(memAllocationTemplateCommWithKey).
-		Sells(clusterTemplateComm)
-
-	return nodeSupplyChainNodeBuilder.Create()
-}
-
 func (f *SupplyChainFactory) buildPodSupplyBuilder() (*proto.TemplateDTO, error) {
-	// Pod supply chain node builder
 	podSupplyChainNodeBuilder := supplychain.NewSupplyChainNodeBuilder(proto.EntityDTO_CONTAINER_POD)
 	podSupplyChainNodeBuilder = podSupplyChainNodeBuilder.
-		Sells(vCpuTemplateComm).
+		Sells(vCpuTemplateComm). //sells to containers
 		Sells(vMemTemplateComm).
 		Sells(vmpmAccessTemplateComm).
 		Provider(proto.EntityDTO_VIRTUAL_MACHINE, proto.Provider_HOSTING).
-		// TODO we will re-include provisioned commodities bought by pod later.
-		//Buys(cpuProvisionedTemplateComm).
-		//Buys(memProvisionedTemplateComm).
-		Buys(clusterTemplateComm).
+		Buys(vCpuTemplateComm).
+		Buys(vMemTemplateComm).
```

Comment:
Pod also buys `Cluster` and `VMPM_ACCESS` commodities. Is it better to only define these two commodities in the `ExternalLink`, or also defined these two here?